docs/commands/README.md

```diff
@@ -326,7 +326,7 @@ USAGE
   $ fluence dependency install [--no-input] [--force]
 
 FLAGS
-  --force     Force install even if the dependency/dependencies is/are already installed
+  --force     Force install using cargo even if the dependency/dependencies is/are already installed
```

Comment:
why cargo? `fluence dep i` is not cargo specific, right?

Reply:
exactly. Thanks for catching that